Tests/test_content.py

```diff
@@ -346,19 +346,19 @@ def get_server_numeric_version(ami_env, is_local_run=False):
 
 
 def extract_server_numeric_version(instances_ami_name, default_version):
-    # regex doesn't catch Server Master execution
-    extracted_version = re.findall(r'Demisto-(?:Circle-CI|Marketplace)-Content-AMI-[A-Za-z]*[-_](\d[._]\d)-[\d]{5}',
-                                   instances_ami_name)
-    extracted_version = [match.replace('_', '.') for match in extracted_version]
+    try:
+        server_numeric_version = re.match(
+            r'server-image-(?:ga-)?(?P<version>[a-z0-9\-]+)-(?P<build_number>\d+)-(?P<creation_date>\d{4}-\d{2}-\d{2})',
+            instances_ami_name
+        ).group('version')
+    except (AttributeError, IndexError):
+        return default_version
 
-    if extracted_version:
-        server_numeric_version = extracted_version[0]
+    if server_numeric_version == 'master':
+        logging.info('Server version: Master')
+        return default_version
```

Comment:
Didn't we have a discussion about changing this to use a dedicated field for the server version?